images/Dockerfile

```diff
@@ -92,13 +92,16 @@ RUN cd /tmp && \
     mv ks_0.8.0_linux_amd64/ks /usr/local/bin && \
     chmod a+x /usr/local/bin/ks
 
-COPY ./requirements.txt /tmp/requirements.txt
+COPY ./Pipfile ./Pipfile.lock /tmp/
```

Comment:
Does this get used anywhere?

Reply:
it is used in https://github.com/kubeflow/testing/pull/36/files#diff-475e714b0c3678141c59dab9961e8841R103